en/snap/integration-guide.md

```diff
@@ -42,45 +36,53 @@ The overall Snap end-to-end payment proccess can be illustrated in following seq
+| Element        | Description                                                  | Requirement |
+| -------------- | ------------------------------------------------------------ | ----------- |
+| `Server Key`   | The server key. For more details, refer to [Retrieving API Access Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys) | Required    |
+| `order_id`     | Unique transaction order ID, defined from your side. One ID could be used only once for the order of the material. Allowed character are Alphanumeric, dash(-), underscore(_), tilde (~), and dot (.) String, max 50. | Required    |
+| `gross_amount` | Total amount of transaction, defined from your side. Integer. | Required    |
 
-#### Request Details
-Type | Value
---- | ---
-HTTP Method | `POST`
-API endpoint (Sandbox) | `https://app.sandbox.midtrans.com/snap/v1/transactions`
-API endpoint (Production) | `https://app.midtrans.com/snap/v1/transactions`
+#### Endpoints
+
+| Environment | Method | URL                                                     |
+| ----------- | ------ | ------------------------------------------------------- |
+| Sandbox     | POST   | `https://app.sandbox.midtrans.com/snap/v1/transactions` |
+| Production  | POST   | `https://app.midtrans.com/snap/v1/transactions`         |
 
 #### HTTP Headers
-```
-Accept: application/json
-Content-Type: application/json
-Authorization: Basic AUTH_STRING
-```
+
+| Header Name   | Description                                            | Required | Values                |
+| ------------- | ------------------------------------------------------ | -------- | --------------------- |
+| Accept        | The format of the data to be returned.                 | Required | application/json      |
+| Content-Type  | The format of the data to be posted.                   | Required | application/json      |
+| Authorization | The authentication method used to access the resource. | Required | Basic **AUTH_STRING** |
 
 **AUTH_STRING**: Base64(`ServerKey + :`)
 
-?> Midtrans API validates HTTP request by using Basic Authentication method. The username is your Server Key while the password is empty. The authorization header value is represented by AUTH_STRING. AUTH_STRING is base-64 encoded string of your username & password separated by **:** (colon symbol). [Follow this reference for more detail about API header](/en/technical-reference/api-header.md).
+?> Midtrans API validates HTTP request by using Basic Authentication method. The username is your Server Key while the password is empty. The authorization header value is represented by AUTH_STRING. AUTH_STRING is base-64 encoded string of your username and password separated by colon symbol (**:**). For more details, refer to [ API Authorization and Headers](https://docs.midtrans.com/en/technical-reference/api-header).
```

Comment:
Do I have to explain this again?

<img width="800" alt="Screen Shot 2020-11-20 at 12 15 02" src="https://user-images.githubusercontent.com/13027142/99762029-21c34c00-2b2a-11eb-958a-95fc21653704.png">